Guesser/DoctrineFieldGuesser.php

```diff
@@ -62,7 +62,8 @@ public function getAllFields($class)
     public function getManyToMany($model, $fieldPath)
     {
         $resolved = $this->resolveRelatedField($model, $fieldPath);
-        return !$this->getMetadatas($resolved['class'])->isAssociationWithSingleJoinColumn($resolved['field']);
+        $metadata = $this->getMetadatas($resolved['class']);
+        return $metadata->hasAssociation($resolved['field']) && !$metadata->isAssociationWithSingleJoinColumn($resolved['field']);
```

Comment:
It is not BC break - it is a bugfix. Currently, a wrong code is generated, that is why I have submitted this patch. 

Per example, for all non-relation columns you would get this code:

    protected function filterColumnName($queryFilter, $value)
    {
        $queryFilter->addXXXFilter('columnName', $value, 1);
    }

and if you have date/datetime field -> you are in trouble, because "addDateFilter" as third parameter has date format as string.

Probably no one noticed a bug so far, but it is a bug.